browser/js/components/Comment/actions.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+export const SELECT_COMMENT = Symbol('SELECT_COMMENT')
+export const COMMENT_EDIT_START = Symbol('COMMENT_EDIT_START');
```

Comment:
It can be a neat way of ensuring not only uniqueness, but also that only something with access to the symbol can actually utilize it. In this case that means something would need to import these event string symbols in order to trigger or listen for the event. Honestly in my opinion it's a very minor improvement here, it's just something I thought I'd mention.